lib/git_worktree.rb

```diff
@@ -215,14 +217,14 @@ def checkout(target_directory)
     @repo.checkout_tree(tree, :target_directory => target_directory, :strategy => :force)
   end
 
-  def with_credential_options
+  def with_remote_options
```

Comment:
Pretty sure this was the case, but this is something that @Fryguy recently changed right?

Just want to make sure nothing else is using this method.  Which since it is so new, that shouldn't be the case, but it is a public method so just checking off that box.

Reply:
Yeah, I added this for building credential options in a common way, and it is only used internally aside from specs (not sure why I didn't make it private)

As discussed with @carbonin in https://github.com/ManageIQ/manageiq/pull/19132#discussion_r313062869, I realized that method didn't *need* to be credentials oriented only, and in adding more connection options, we felt better to rename the method to make it a bit broader.

Reply:
Cool thanks, missed that convo on my first pass.  Make sense and figured that was the case.